extensions/amp-analytics/amp-analytics.md

```diff
@@ -173,6 +173,87 @@ In this example, we specify the `config` attribute to load the configuration dat
 <amp-analytics config="https://example.com/analytics.account.config.json">
 ```
 
+#### Configuration Rewriter
+
+The configuration rewriter feature is designed to allow analytics providers to dynamically rewrite a provided configuration. This is similar to the remote configuration feature but additionally includes any user-provided configuration in the request made to the sever. This currently can only be enabled by an analytics vendor.
+
+An analytics vendor will specify a `configRewriter` property with a server url.
+```js
+export const VENDOR_ANALYTICS_CONFIG = {
+    ...
+    'configRewriter': {
+      'url': 'https://www.vendor.com/amp-config-rewriter',
+    },
+    ...
+}
+```
+
+The runtime will fetch the remote configuration (if given) and merge it with the configuration inlined in the document, before sending a request to `configRewriter` endpoint given by the vendor. The vendor can then use this data server side to construct and return a new rewritten configuration.
+
+The runtime will then merge all these configurations in order to determine the final configuration. This will happen in this order of precedence (highest to lowest):
+1. Rewritten Configuration
+1. Inlined Configuration
+1. Vendor defined configuration
+
+##### Variable Groups
+
+Variable Groups is a feature that allows analytics providers to provide a predefined set of variables that can easily be enabled by an user. These variables will then be resolved and sent along to the specified `configRewriter` endpoint.
+
+To set up this feature the analytics provider should create a new `varGroups` object inside of the `configRewriter` configuration. Inside this object you may include any named variable groups you might want enabled. All of the variables supported by [AMP HTML Substitutions Guide](../../spec/amp-var-substitutions.md) can be used. A _important_ note is that the `${varName}` variants will not work.
```

Comment:
"To set up this feature the analytics provider should create a new `varGroups` object inside of the `configRewriter` configuration. Inside this object you may include any named variable groups you might want enabled."

to

"Use the analytics provider's created `varGroups` objects inside the `configRewriter` configuration. This object will include any named variables groups you want enabled."

Reply:
I think this is a different meaning, but I agree it could be improved. The pub will need to create a `varGroups`  configuration on their page to opt in to the predefined `varGroups` given by an analytics vendor. Does that make sense?

Reply:
Could you provide a code sample for it?

Reply:
I think this is a closer:
"To set up this feature the analytics provider should create a new `varGroups` object inside of the `configRewriter` configuration. Inside this object you may include any named variable groups you might want enabled."

to

"Analytics providers need to create a new `varGroups` object inside of the `configRewriter` configuration to enable this feature. Publishers can then include any named analytic provider created `varGroups` they wish to enable in their analytics configuration."

Reply:
Done.